app-admin/pwman3/pwman3-0.10.0-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite?"
+inherit distutils-r1
+
+DESCRIPTION="A lightweight password-manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io/pwman3/"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="mongodb mysql postgres +sqlite test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="
```

Comment:
Use `COMMON_DEPEND` instead.

Reply:
I think this is now fixed. Can you take a look?